index.js

```diff
@@ -1,14 +1,14 @@
 var ApiClient = require('./src/apiclient');
 
-function TelegramBotClient (token, promise) {
+function TelegramBotClient (token, params, promise) {
```

Comment:
We will now need to pass through `params' when creating new `TelegramBotClient` instances further down in the code.